boot/freeldr/freeldr/CMakeLists.txt

```diff
@@ -17,10 +17,17 @@ endif()
 spec2def(freeldr_pe.exe freeldr.spec)
 
 if(ARCH STREQUAL "i386")
-    CreateBootSectorTarget(frldr16
-        ${CMAKE_CURRENT_SOURCE_DIR}/arch/realmode/i386.S
-        ${CMAKE_CURRENT_BINARY_DIR}/frldr16.bin
-        F800)
+    if(SARCH STREQUAL "pc98") # FIXME: Use single i386.S for both PC and PC-98 platforms
```

Comment:
What are the actual differences between i386.S and i386_pc98.S ?

Reply:
The only difference being a single line from includes section.

Reply:
If so, I think you can then just keep the i386.S file and use that construct:
```c
#if defined(SARCH_PC98)
#include "helpers_for_pc98.inc"
#else
#include "helpers_for_pc.inc"
#endif
```

Reply:
Sorry, I don't know how to add this define into ASM files.

Reply:
Also see changes in `int386.inc`. I missed the define there.

Reply:
OK now with latest master the `SARCH_xxx` defines should be visible within ASM files too.

Reply:
Thanks, updated.